src/navigators/Main.tsx

```diff
@@ -1,14 +1,27 @@
 import React from 'react';
-import { Home } from '../screens';
 import { createStackNavigator } from '@react-navigation/stack';
+import PhoneAuth from '../screens/auth/phone-auth';
+import OTPVerify from '../screens/auth/otp-verify';
+import AuthenticatedStack from './authenticated';
+import { useAuthContext } from '../contexts';
+import { MainParamsList } from '../../@types/navigation';
 
-const Stack = createStackNavigator();
+const Stack = createStackNavigator<MainParamsList>();
```

Comment:
This code is being repeated in `src/navigators/main.tsx`. Can tell - What's the difference between the two files and can we remove the duplicate?

Reply:
removed the duplicate version of main and application file